src/cli/bin.js

```diff
@@ -16,6 +16,12 @@ updateNotifier({
 const cli = yargs
   .commandDir('commands')
   .demandCommand(1)
+  .fail((msg, err, yargs) => {
+    if (err) throw err // preserve stack
+    console.error(yargs.help())
+    process.exit(1)
+  })
+
```

Comment:
The Travis tests are failing, because the linting is failing with this added empty line :)